src/cephadm/cephadm

```diff
@@ -7577,6 +7584,10 @@ def _get_parser():
         '--image',
         help='container image. Can also be set via the "CEPHADM_IMAGE" '
         'env var')
+    parser.add_argument(
+        '--skip-tls-pull',
+        action='store_true',
+        help='do not use tls when pulling image')
```

Comment:
this is unfortunately a bit deceiving to users, as this only works for tiny toy environments. If you want to advertise it in the cephadm CLI, you'll need to make this actually work reliably. I.e. this needs to end up in very similar places as the `container-init` flag. 

1. It needs to be another MGR module option
2. and passed though all `run_cephadm` calls
3. and written to all unit.run files

As users might remove all images and rely on the systemd unit files to re-pull the images on the hosts